### PR TITLE
[conversions] bug fix in GetFrame

### PR DESCRIPTION
**Pull request by cmower**
Fix for #648 
<!--
# Checklist

- code formatting: run `find -name '*.cpp' -o -name '*.h' -o -name '*.hpp' | xargs clang-format-3.9 -style=file -i` in the root directory

- add unit test(s)

- ensure tests build and check results: run `catkin run_tests exotica && catkin_test_results`

--!>

### CODE REVIEW

File: exotica_core/src/tools/conversions.cpp [hunk 1]
```diff
@@ -93,7 +93,7 @@ Eigen::MatrixXd GetFrame(const KDL::Frame& val)
     ret(0, 3) = val.p(0);
     ret(1, 3) = val.p(1);
     ret(2, 3) = val.p(2);
-    ret.block(0, 0, 3, 3) = Eigen::Map<const Eigen::Matrix3d>(val.M.data);
+    ret.block(0, 0, 3, 3) = Eigen::Map<const Eigen::Matrix3d>(val.M.data).transpose();
```
Comment (wxmerkt) [inline exotica_core/src/tools/conversions.cpp]: Cf. https://github.com/ipab-slmc/exotica/issues/648#issuecomment-519654484

File: exotica_core/src/tools/conversions.cpp [hunk 2]
```diff
@@ -89,21 +90,19 @@ KDL::Frame GetFrameFromMatrix(Eigen::MatrixXdRefConst val)
 
 Eigen::MatrixXd GetFrame(const KDL::Frame& val)
 {
-    Eigen::MatrixXd ret = Eigen::MatrixXd::Identity(4, 4);
-    ret(0, 3) = val.p(0);
-    ret(1, 3) = val.p(1);
-    ret(2, 3) = val.p(2);
-    ret.block(0, 0, 3, 3) = Eigen::Map<const Eigen::Matrix3d>(val.M.data).transpose();
-    return ret;
+    Eigen::Affine3d ret;
```
Comment (wxmerkt) [inline exotica_core/src/tools/conversions.cpp]: Please use Isometry, not Affine.

Comment (cmower) [reply exotica_core/src/tools/conversions.cpp]: Any reason for this?

Comment (wxmerkt) [reply exotica_core/src/tools/conversions.cpp]: Yes. If you would do an inverse on the Affine, it runs SVD which is expensive. Rigid body motions do not do shearing, scaling etc. so no need to have. Isometry is much cheaper with the inverse as it's direct.

**Comment by cmower** [comment]
Yes, all returning correct values. No problem!

**Pull request merged**